kubernetes/README.md

```diff
@@ -0,0 +1,62 @@
+# Kubernetes Helm Chart
+
+
+
+**Not ready for customers. Not ready for production.** 
```

Comment:
Will this be true when it's merged?

Reply:
Should we test this Chart on AWS before merging?

Reply:
We should, for purposes of claiming suitable persistent volumes.